package.json

```diff
@@ -20,7 +20,8 @@
     "@testing-library/dom": "10.1.0",
     "@types/react": "18.2.0",
     "@types/react-dom": "18.2.0",
-    "jackspeak": "2.1.1"
+    "jackspeak": "2.1.1",
+    "@leafygreen-ui/hooks": "8.1.3"
```

Comment:
I would prefer if we get LG to fix this on their end instead of pinning the dependency since we would likely forget about this or run into unpredictable behavior if LG updates this in the future.

Reply:
I think we can just ticket this and track it on our side. It's been broken since August 27th, I think we should patch it